pages/Crewmates.tsx

```diff
@@ -1,61 +1,95 @@
 import React from 'react';
 import Head from 'next/head';
 
-// Components
-import Article from '../components/article';
-
 // Images
 const Founder = '/images/1.jpg';
+const icon = '/images/icon.jpg';
 
 const CrewMates = () => (
-    <div className='  bg-dark-primary text-white'>
+    <div className='bg-dark-primary text-white'>
+
         <Head>
             <title>CrewMates </title>
 
         </Head>
+        <img className='w-20 h-20  relative object-cover inline-block ' src={icon} width='20%' />
+        <div className='inline-block text-white text-3xl font-bold leading-none mb-6 p-5'>CrewMates</div>
 
-        <div text-white text-6xl font-bold leading-none mb-6>CrewMates</div>
-        <div className='w-11/12 underline'>
+        <div className='w-11/12 my-6 p-12'>
 
-            <ul>
-                <li className='inline'><a className='hover:underline focus:shadow-outline' href="#">Core Team</a></li>
-                <li className='inline'><a className='hover:underline focus:shadow-outline' href="#">Dev Team</a></li>
-                <li className='inline'><a className='hover:underline focus:shadow-outline' href="#">Authors</a></li>
+            <ul className='border-b-2'>
+                <li className='inline p-10 m-4 text-2xl'><a className='hover:underline ' href="#">Core Team</a></li>
+                <li className='inline p-10 m-12 text-2xl'><a className='hover:underline ' href="#">Dev Team</a></li>
+                <li className='inline p-10 m-12 text-2xl'><a className='hover:underline ' href="#">Authors</a></li>
             </ul>
         </div>
 
-        <div>
-            <img className='w-full h-56 relative object-cover' src={Founder} width='80%' />
-            <p className='text-yellow-500 absolute bottom-0 left-0 top-0 invisible hover:visible text-3xl'>SHIVAM SINGHAL FOUNDER</p>
-            <p className='text-gray-300 font-semibold leading-normal'>
-                this is my page This is a test file, test check tezzos common,
-                hail hydra This is a test file, test check tezzos common, hail
-                hydra This is a test file, test check tezzos common, hail hydra
-             </p>
+        <div className='box-border relative inline-block w-4/12 h-auto'>
```

Comment:
@shivanigupta71299  This div (parent of image) has width w-4/12, remove this and your image will acquire it's specified size. OR give this div width 100% if the former doesn't work.